conf/openlibrary.yml

```diff
@@ -77,9 +77,6 @@ features:
     support: admin
     undo: enabled
     upstream: enabled
-    user_metadata:
-        filter: usergroup
-        usergroup: /usergroup/beta-testers
```

Comment:
Started this work by adding a new feature to the yml.  Discovered that this wasn't needed, and it creates issues on environments that do not already have the usergroup in the DB (mainly local dev environments).

While doing all of that, I noticed that this book tags-related config is no longer needed (and never really was...).